benchmarks/grain_size_pinned_state/dynamic_grain_size_pinned_state.ipynb

```diff
@@ -0,0 +1,163 @@
+{
+  "cells": [
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "vA8IKpRQToxA"
+      },
+      "source": [
+        "This notebook reports on the test results for test `grain_size_strain_pinned.prm` and allows to compare the test against analytical results. The test models a small volume in simple shear flow with constant strain rate, and a grain size that is not at its equilibrium value. We can compute how much the grain size should change according to the terms given in Eq. 8 of Mulyukova and Bercovici (2018):\n",
+        "\n",
+        "\\begin{align}\n",
+        "\\text{grain\\_growth} = \\frac {3 \\phi_1\\phi_2 k_g \\sqrt h_g^{-q}}{(q R^{q-1})} \\exp(\\frac{-E_g}{R_gT}) \\\\\n",
```

Comment:
I do not think you need the double \ before the underscore. At least it now has lots of `\_` in the notebook.